package.json

```diff
@@ -15,24 +22,23 @@
     "node": ">=16.0.0"
   },
   "dependencies": {
-    "axios": "^0.21.1"
+    "axios": "^0.27.2"
   },
   "devDependencies": {
-    "@types/node": "^16.3.2",
+    "@osskit/eslint-config": "^1.0.3",
+    "@osskit/prettier-config": "^0.0.1",
+    "@types/node": "^18.0.0",
```

Comment:
Not directly related to ESM, but might consider upgrading node 16 to 18 everywhere